omas/tests/test_omas_physics.py

```diff
@@ -187,6 +187,7 @@ def CPC(ods, kw=jdef, should_RE=False, should_AE=False, warn=False):
 
     def test_sumary_global_quantities(self):
         ods = ODS().sample()
+        ods['summary'] = ODS()
```

Comment:
@TimSlendebroek why is this needed?

Reply:
Well the sample doesn't have summary, which makes triggering ods.summary_global_quanties() not possible

Reply:
Can we generalize this fix and make it so that `summary_global_quantities` adds `ods['summary']` if that's not present?

Reply:
Yeah that's fair good point!